disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -32,6 +33,7 @@ const (
 	OperatorResponseIndexName  = "OperatorResponseIndex"
 	RequestedAtIndexName       = "RequestedAtIndex"
 	AttestedAtIndexName        = "AttestedAtAIndex"
+	AccountBlobIndexName       = "AccountBlobIndex"
```

Comment:
Needs an infrastructure change to add this index?

Reply:
yea, having a local change which will be sent out later